recipe/meta.yaml

```diff
@@ -21,12 +21,12 @@ outputs:
     build:
       noarch: python
       number: {{ build }}
-      string: "pyh{{ PKG_HASH }}_{{ build }}_{{ qt_bindings }}"
+      string: pyh{{ PKG_HASH }}_{{ build }}_{{ qt_bindings }}
```

Comment:
@jaimergp @goanpeca this line was auto-updated by the bot — not sure whether to keep it like this or revert the change? Other than this, this is ready to merge imho.